vscode/microsoft-kiota/src/steps.ts

```diff
@@ -316,9 +318,10 @@ export async function generateSteps(existingConfiguration: Partial<GenerateState
                 validate: validateIsNotEmpty,
                 shouldResume: shouldResume
             });
-            pluginTypes.label === 'API Plugin' ? state.pluginTypes = ['ApiPlugin'] : state.pluginTypes = ['OpenAI'];
+            pluginTypes.label === 'API Manifest' ? state.pluginTypes = ['ApiManifest'] : state.pluginTypes = ['OpenAI'];
+            pluginTypes.label === 'API Manifest' ? state.generationType = 'apimanifest' : state.generationType = 'plugin';
         }
-        return (input: MultiStepInput) => inputPluginOutputPath(input, state);
+        return (input: MultiStepInput) => inputManifestName(input, state);
```

Comment:
When user selects label 'OpenAI' do we still want next question to be "'Choose a name for the manifest" as specified in the inputManifestName? From AC it looks like next step after user chooses `OpenAI` would be to direct to the `inputPluginOutputPath`.

Reply:
We'll follow the AC. As it has recently been clarified